hibernate-core/src/main/java/org/hibernate/query/sqm/tree/domain/SqmBasicValuedSimplePath.java

```diff
@@ -45,12 +45,13 @@ public SqmBasicValuedSimplePath(
 
 	@Override
 	public SqmBasicValuedSimplePath<T> copy(SqmCopyContext context) {
+		final SqmPath<?> lhsCopy = getLhs().copy( context );
```

Comment:
Please revert the changes here and instead remove the copying of the `reusablePaths` in `org.hibernate.query.sqm.tree.domain.AbstractSqmPath#copyTo`.

Reply:
Still, please revert the change in this class.

Reply:
Sorry, forgot that.